main.py

```diff
@@ -2,6 +2,30 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
```

Comment:
The genre 'Dramma' seems to be a typo. It is later corrected to 'Drama', but it would be more efficient to use the correct spelling initially.